uap_tracker/utils.py

```diff
@@ -47,13 +47,18 @@ def get_writer(output_filename, width, height):
 # Utility function to convert jey points in to a bounding box
 # The bounding box is used for track validation (if enabled) and will be displayed by the visualiser
 # as it tracks a point of interest (blob) on the frame
-def kp_to_bbox(kp):
+def kp_to_bboxOld(kp):
     (x, y) = kp.pt
     size = kp.size
     scale = 6
     #print(f'kp_to_bbox x, y:{(x, y)}, size:{size}, scale:{scale}, new size:{scale * size}')
     return (int(x - scale * size / 2), int(y - scale * size / 2), int(scale * size), int(scale * size))
 
+# Since the new blob detector already returns the bounding box, this just return the same
+# it is just kept here so we don't need to edit the whole code
+def kp_to_bbox(kp):
```

Comment:
I think this is a little confusing :-) Are you getting a bbox back fro your blob detector? Do you need to call the kp_to_bbox method if its just ultimately pass through?